src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/SealightsCLIBuildStep.java

```diff
@@ -243,18 +243,16 @@ public String getDisplayName() {
         }
     }
 
-    private CLIRunner createCLIRunner(CommandMode commandMode){
+    public CLIRunner createCLIRunner(CommandMode commandMode){
         if(commandMode instanceof CommandMode.ConfigView){
             CommandMode.ConfigView configView=(CommandMode.ConfigView) commandMode;
             return new CLIRunner(configView.getBuildSessionId(),configView.getAppName(),configView.getBranchName(),
                     configView.getBuildName(),configView.getAdditionalArguments(),null);
         }else {
             Properties properties = PropertiesUtils.toProperties(commandMode.getAdditionalArguments());
-            String appName = (properties.get("appname")!= null)? properties.get("appname").toString():null;
-            String branchName= (properties.get("branchname")!= null)? properties.get("branchname").toString():null;
             String labId= (properties.get("labid")!= null)? properties.get("labid").toString():null;
-            CommandBuildName buildName = new CommandBuildName.ManualBuildName((String) properties.get("buildname"));
-            return new CLIRunner(commandMode.getBuildSessionId(),appName,branchName,buildName,commandMode.getAdditionalArguments(),labId);
+            CommandBuildName buildName = new CommandBuildName.EmptyBuildName();
```

Comment:
Why do we need to pass empty build name?

Reply:
in config and prConfig commands we send a build name but for commands that use bsid we don't need it.
We have a few types of build (manual/latest and few more) one of them is called 'Empty Build' leave the buildname  field empty, this let us avoid null handling